opus/application/static_media/js/browse.js

```diff
@@ -1486,8 +1498,12 @@ var o_browse = {
                             opus.prefs[startObsLabel] = scrollbarObsNum;
                         }
                     }
-                    let path = o_browse.getDataURL(view, obsNum, customizedLimitNum);
-                    return path;
+                    // if totalObsCount is not yet defined, we still need to init the infinite scroll...
+                    if (viewNamespace.totalObsCount === undefined || obsNum <= viewNamespace.totalObsCount) {
+                        let path = o_browse.getDataURL(view, obsNum, customizedLimitNum);
+                        return path;
+                    }
+                    // returning no value indicates end of infinite scroll...
```

Comment:
When you hit the end of the observations, the console shows this error:
`infinite-scroll.pkgd.js:1265 GET http://127.0.0.1:8000/undefined 404 (Not Found)`
From that point on infinite scroll fails to work. I think Dave discovered that the returning of "undefined" doesn't work as documented. That's why we were still retrieving extra data even at the end, causing multiple API calls. He hadn't figured out a way around it. The fix you put in breaks scrolling entirely. It would be better to just have the multiple API calls for now.

Reply:
hmm.  I can see that this funciton from infinite-scroll:
function request( url, responseType, onLoad, onError, onLast ) {
  var req = new XMLHttpRequest();
  req.open( 'GET', url, true );
appears to not bail when url is empty, thus being an issue.
I have posted a note to GIT regarding this; i can remove and note but this will also cause side effects.